src/containers/Home/Home.js

```diff
@@ -78,6 +72,21 @@ class Home extends Component {
     this.props.getAllCategory();
   }
 
+  /**
+   * @returns {func} article
+   * @memberof Home
+   */
+  componentDidUpdate() {
+    if (document.querySelector('.carousel') && !this.state.sliderStarted) {
```

Comment:
can this moved into a method, then call the method here

Reply:
fixed